pjlib/src/pj/ssl_sock_imp_common.c

```diff
@@ -1134,7 +1134,6 @@ static pj_bool_t ssock_on_accept_complete (pj_ssl_sock_t *ssock_parent,
                                                    ssock->param.grp_lock);
         if (status != PJ_SUCCESS) {
             ssock->timer.id = TIMER_NONE;
-            status = PJ_SUCCESS;
```

Comment:
Won't this change the behavior (more than fixing a warning)?
The original code seems to start the handshake without a timeout when the timer scheduling fails?
Now, it will trigger handshake failure?